.github/workflows/qa.yml

```diff
@@ -68,6 +68,54 @@ jobs:
       - name: Assert
         run: |
           ./test/assertFileContains ./output.properties "sonar.projectBaseDir=.*/baseDir"
+  scannerVersionTest:
```

Comment:
This test was missing. While it is not strictly in scope for the change, but helped as intermediate testing step to come up with a valid test for `scannerBinariesUrl`.